schema.graphql

```diff
@@ -191,4 +202,13 @@ type Activity @entity {
   auctionCollateral: ERC721Token
   auctionTokenId: BigInt
   auctionEndPrice: BigInt
+  position: Position
+  totalLiquidityAdded: BigInt
+  totalLiquidity0: BigInt
```

Comment:
I think totalLiquidity0 is kinda confusing: this is cumulative token0 right? also I think we only need to store the cumulative, do we need to know how much was added here? Was thinking we could just look at the prev cumulative and know the diff

Reply:
yeah we need to know how much was added for the activity feed